script/update-cli-version

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+main()
+{
+    local version=$(git describe --tags)
+    /usr/libexec/PlistBuddy -c "Set CFBundleShortVersionString $version" Source/carthage/Info.plist
```

Comment:
`Source/carthage/Info.plist` should probably be relative to this script's path.
I'd recommend doing:
```bash
DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
```

Then getting the path like so:
```bash
"$DIR/../Source/carthage/Info.plist"
```

Make sure you add quotes though in case there are spaces in the path!